lib/populator.js

```diff
@@ -60,7 +70,7 @@ Populator.prototype._getTrackedEntityAttributeType = function (trackedEntityAttr
   }
   this.emit('getTrackedEntityAttributeType', trackedEntityAttributeID);
   Request.get({
-    url: URL.resolve(this._options.url, 'api/trackedEntityAttributes/' + trackedEntityAttributeID),
+    url: URL.resolve(this._options.url, createDhisUrl(this._options.dhisVersion, 'trackedEntityAttributes/') + trackedEntityAttributeID),
```

Comment:
The ID should be included the the string passed to `createDhisUrl`. That way if we need to add query parameters or anything like that in future it will work.